marimo/_plugins/stateless/sidebar.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Marimo. All rights reserved.
+from __future__ import annotations
+
+from typing import Optional
+
+from marimo._output import md
+from marimo._output.formatting import as_html
+from marimo._output.hypertext import Html
+from marimo._output.rich_help import mddoc
+from marimo._plugins.core.web_component import build_stateless_plugin
+from marimo._plugins.stateless.flex import vstack
+
+
+@mddoc
+def sidebar(
+    element: object,
```

Comment:
For consistency with (most) of the other layout functions:

```suggestion
    item: object,
```